instrumentation/elasticsearch/elasticsearch-transport-5.3/src/test/groovy/springdata/Elasticsearch53SpringTemplateTest.groovy

```diff
@@ -43,290 +43,290 @@ import static io.opentelemetry.trace.Span.Kind.CLIENT
 import static org.elasticsearch.cluster.ClusterName.CLUSTER_NAME_SETTING
 
 class Elasticsearch53SpringTemplateTest extends AgentTestRunner {
-  public static final long TIMEOUT = 10000; // 10 seconds
-
-  // Some ES actions are not caused by clients and seem to just happen from time to time.
-  // We will just ignore these actions in traces.
-  // TODO: check if other ES tests need this protection and potentially pull this into global class
-  public static final Set<String> IGNORED_ACTIONS = ImmutableSet.of("NodesStatsAction", "IndicesStatsAction")
-
-  @Shared
-  Node testNode
-  @Shared
-  File esWorkingDir
-  @Shared
-  String clusterName = UUID.randomUUID().toString()
-
-  @Shared
-  ElasticsearchTemplate template
-
-  def setupSpec() {
-
-    esWorkingDir = File.createTempDir("test-es-working-dir-", "")
-    esWorkingDir.deleteOnExit()
-    println "ES work dir: $esWorkingDir"
-
-    def settings = Settings.builder()
-      .put("path.home", esWorkingDir.path)
-    // Since we use listeners to close spans this should make our span closing deterministic which is good for tests
-      .put("thread_pool.listener.size", 1)
-      .put("transport.type", "netty3")
-      .put("http.type", "netty3")
-      .put(CLUSTER_NAME_SETTING.getKey(), clusterName)
-      .build()
-    testNode = new Node(new Environment(InternalSettingsPreparer.prepareSettings(settings)), [Netty3Plugin])
-    testNode.start()
-    runUnderTrace("setup") {
-      // this may potentially create multiple requests and therefore multiple spans, so we wrap this call
-      // into a top level trace to get exactly one trace in the result.
-      testNode.client().admin().cluster().prepareHealth().setWaitForYellowStatus().execute().actionGet(TIMEOUT)
-    }
-    TEST_WRITER.waitForTraces(1)
-
-    template = new ElasticsearchTemplate(testNode.client())
-  }
+    public static final long TIMEOUT = 10000; // 10 seconds
```

Comment:
can you try reformatting this file in Intellij? i'm not sure why it's not using our `.editorconfig`